src/Reveal.Sdk.Dom.Tests/Visualizations/Extensions/IFinanceExtensionsFixture.cs

```diff
@@ -0,0 +1,217 @@
+﻿using Reveal.Sdk.Dom.Visualizations;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace Reveal.Sdk.Dom.Tests.Visualizations.Extensions
+{
+    public class IFinanceExtensionsFixture
+    {
+        [Fact]
+        public void SetOpen_UpdateOpen_WithFieldName()
+        {
+            // Arrange
+            var visualization = new MockIFiananceVisualization() { Opens = new List<MeasureColumn>() { new MeasureColumn() { DataField = new NumberDataField("InitialField") } } };
+            var fieldName = "TestFieldName";
+            var field = new NumberDataField(fieldName);
+            field.Formatting = new NumberFormatting()
+            {
+                DecimalDigits = 0,
+                ShowGroupingSeparator = true
+            };
+            var expectedOpen = new List<MeasureColumn> { new MeasureColumn() { DataField = field } };
+
+            // Act
+            visualization.SetOpen(fieldName);
+
+            // Assert
+            Assert.Equivalent(expectedOpen, visualization.Opens);
+        }
+
+        [Theory]
+        [InlineData(true)]
+        [InlineData(false)]
+        public void SetOpen_UpdateOpen_WithNumberDataField(bool isFormattingNull)
+        {
+            // Arrange
+            var visualization = new MockIFiananceVisualization() { Opens = new List<MeasureColumn>() { new MeasureColumn() { DataField = new NumberDataField("InitialField") } } };
+
+            var fieldFormat = isFormattingNull ? null : new NumberFormatting()
+            {
+                DecimalDigits = 2,
+                ShowGroupingSeparator = false
+            };
+            var field = new NumberDataField("TestField")
+            {
+                Formatting = fieldFormat
+            };
+
+            var expectedOpen = new List<MeasureColumn> { new MeasureColumn() { DataField = field } };
+
+            // Act
+            visualization.SetOpen(field);
+
+            // Assert
+            Assert.Equivalent(expectedOpen, visualization.Opens);

```

Comment:
Please add assert for number formating in both cases

Reply:
Hi @hainv-ohio 
`Assert.Equivalent` will check the 2 objects and child objects inside them. So I think that no need to check number formatting. It's checked with ` Assert.Equivalent(expectedOpen, visualization.Opens);`